src/workflows/decryptVerifyAccessToken.ts

```diff
@@ -0,0 +1,40 @@
+import type { AccessToken } from "@src/@types/oauth2/OAuth2Tokens";
+import { oauth2Client, config } from "@src/config";
+import {
+  ConnectIsAccessTokenSignedMustBeBoolean,
```

Comment:
```suggestion
  ConnectIsAccessTokenSignedMustBeBoolean,
```

This is quite confusing 😅.

Reply:
The error name ?

Reply:
Yes.

Reply:
I wanted to have the real environment variable name in it, but maybe `IsAccessTokenSignedEnvVarMustBeBoolean`is better ? If not, do you have any idea in mind ? 🙂

Reply:
I was confused by the `..Is..Must..`, but it makes sense now because it is the name of the config's field.

Question here is, do we blindly follow naming conventions, or are we going for `EnvVar_IsAccessTokenSigned_MustBeBoolean`, for the sake of clarity.

Any thoughts @Fenntasy?